feed-integration-webapp/src/test/java/org/bahmni/module/feedintegration/integrationtest/BaseIntegrationTest.java

```diff
@@ -0,0 +1,60 @@
+package org.bahmni.module.feedintegration.integrationtest;
+
+import ca.uhn.hl7v2.DefaultHapiContext;
+import ca.uhn.hl7v2.HL7Exception;
+import ca.uhn.hl7v2.HapiContext;
+import ca.uhn.hl7v2.app.HL7Service;
+import ca.uhn.hl7v2.model.Message;
+import ca.uhn.hl7v2.model.v25.message.ORM_O01;
+import ca.uhn.hl7v2.protocol.ReceivingApplication;
+import ca.uhn.hl7v2.protocol.ReceivingApplicationException;
+import org.bahmni.module.feedintegration.CraterAtomfeed;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.runner.RunWith;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.modules.junit4.PowerMockRunnerDelegate;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.test.SpringApplicationConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.net.Inet4Address;
+import java.net.UnknownHostException;
+import java.util.Map;
+
+@RunWith(PowerMockRunner.class)
+@PowerMockRunnerDelegate(SpringJUnit4ClassRunner.class)
+@SpringApplicationConfiguration(classes = CraterAtomfeed.class)
+public abstract class BaseIntegrationTest {
+
+    public static HL7Service modalityStubServer;
+    private static final Logger log = LoggerFactory.getLogger(BaseIntegrationTest.class);
+
+    @Before
+    public void startModalityStubServer() throws InterruptedException, UnknownHostException {
```

Comment:
What is this method for? Modality is PACS terminology. Why is this here?

Reply:
Test was not being used so has been removed.